### ISSUE TITLE
Find in page freeze

### ISSUE THREAD

**Issue by bambooCZ** [nwjs/nw.js#6326]
`webview.find(...)` freezes app then multiple webviews are present and one of them has transferred from `display: none`. It is hard to tell details because NWjs becomes completelly unresponsive...

Chromium 61 behaves exactly the same, Chromium >= 62 is ok.
NWjs 0.27.1 still freezes. Replicable on macOS >= 10.12, Windows >=7. I haven't tested it on Linux.

![screencast](https://user-images.githubusercontent.com/8124126/33901590-9388db2c-df72-11e7-879f-fc0cfc6371b7.gif)

Manual steps:
1. Click "Switch webviews" - this will toggle `display:none` on the webviews
2. Type something into the input - it will call `webview.find(input.value, {}, function(){ ... })` on the currentlly visible webview 

[find-in-page-trouble.zip](https://github.com/nwjs/nw.js/files/1552746/find-in-page-trouble.zip)

**Comment by Christywl** [comment]
I can reproduce this issue on Linux/Windows with nwjs-sdk-v0.27.1.

**Comment by janRucka** [comment]
I was able to reproduce it with Chrome 62 under Win7. 63 is Ok thought. But NWjs27 (as well as NWjs26) is freezing to me.

It seems that it ends up stuck infinitely in this loop:
https://github.com/nwjs/chromium.src/blob/nw27/content/browser/find_request_manager.cc#L632

**Comment by rogerwang** [comment]
The code path is affected by a feature `GuestViewCrossProcessFrames`, which is under field trial control in Chrome (not Chromium).  I guess that's why you two see different results. This feature is disabled in NW but enabled in Chromium.

**Comment by rogerwang** [comment]
Use the following patch seems to fix it. Need to figure out #6171 where we use this as workaround.

```
diff --git a/testing/variations/fieldtrial_testing_config.json b/testing/variations/fieldtrial_testing_config.json
index fb13c42ee5c7..e3b8bd3735fa 100644
--- a/testing/variations/fieldtrial_testing_config.json
+++ b/testing/variations/fieldtrial_testing_config.json
@@ -1420,8 +1420,8 @@
             ],
             "experiments": [
                 {
-                    "name": "Disabled",
-                    "disable_features": [
+                    "name": "Enabled",
+                    "enable_features": [
                         "GuestViewCrossProcessFrames"
                     ]
                 }
```

**Comment by bambooCZ** [comment]
Sufficient solution is to put `"chromium-args": "--enable-features=GuestViewCrossProcessFrames"` into `package.json`.

But do we agree it is only a hotfix? And this issue should be fixed without `chromium-args` magic?

**Comment by rogerwang** [comment]
Could you please file more issues for those and CC #6171 ? I look to enable `GuestViewCrossProcessFrames` by default as upstream is moving towards that.

**Comment by rogerwang** [comment]
I just pushed a real fix for #6171 with `GuestViewCrossProcessFrames` turned on. You could try if it fixes your issues with the next nightly build.

**Comment by rogerwang** [comment]
close as it's fixed by enabling GuestViewCrossProcessFrames
